src/earn/types.ts

```diff
@@ -69,11 +67,11 @@ export interface BeforeDepositAction {
 }
 
 export interface WithdrawAction {
-  name: WithdrawActionName
+  name: EarnActiveAction
   title: string
   details: string
   iconComponent: React.MemoExoticComponent<({ color }: { color: Colors }) => JSX.Element>
   onPress: () => void
 }
 
-export type EarnEnterMode = 'deposit' | 'swap-deposit' | 'withdraw'
+export type EarnActiveAction = 'withdraw' | 'claim-rewards' | 'deposit' | 'swap-deposit' | 'exit'
```

Comment:
if you're changing this to `Action`, can we also change variable names to be `action` instead of `mode`

Reply:
Did the reverse where I kept the variable names mode, but swapped the type name to Mode.